docs/content/doc/advanced/adding-legal-pages.en-us.md

```diff
@@ -0,0 +1,36 @@
+---
+date: "2019-12-28"
+title: "Adding Legal Pages"
+slug: adding-legal-pages
+weight: 9
+toc: false
+draft: false
+menu:
+  sidebar:
+    parent: "advanced"
+    name: "Adding Legal Pages"
+    identifier: "adding-legal-pages"
+    weight: 9
+---
+
+Some jurisdictions (such as EU), requires certain legal pages (e.g. Privacy Policy) to be added to website. Follow these steps to add them to your Gitea instance.
+
+## Getting Pages
+
+Gitea source code ships with sample pages, available in `contrib/legal` directory. Copy them to `custom/public/`. For example, to add Privacy Policy:
+
+```
+cp $GOPATH/src/code.gitea.io/gitea/contrib/legal/privacy.html.sample /path/to/custom/public/
+```
+
+Now you can hack the page to your requirements.
```

Comment:
Even if I like the term `hack`, I would suggest the more common word `edit`.
I would suggest also to use the term `need` since it there is some engagement in it that are not managed by Gitea like:.
```
we will delete your full profile (within reason) within 90 days of your request
``` 
```suggestion
Now you need to edit the page to meet your requirements.
```